app/assets/stylesheets/provider/_theme.scss

```diff
@@ -107,3 +107,4 @@
 @import 'provider/print';
 
 @import 'provider/vertical_nav';
+@import 'provider/patternfly_ie11';
```

Comment:
Do we need to include this in the theme? We should load that file selectively.

Reply:
It has to be here I think, since it affects the whole UI, not only the nav bar.